### ISSUE TITLE
{ content: Number } breaks rendering?

### ISSUE THREAD

**Issue by TuckerWhitehouse** [jamestalmage/cli-table2#11]
Using the object notation for defining cell contents breaks if a number if given and not cast as a string.

```javascript
var Table = require('cli-table2')
var table = new Table()
table.push([{content: 12}]) // Fails
console.log(table.toString())
```
```javascript
var Table = require('cli-table2')
var table = new Table()
table.push([{content: String(12)}]) // Works
console.log(table.toString())
```

**Comment by jamestalmage** [comment]
Fix published in `0.1.8`.

Thanks for the bug report.

**Comment by TuckerWhitehouse** [comment]
Awesome, thanks!
